### ISSUE TITLE
Mapper type 11, memory requirements

### ISSUE THREAD

**Issue by fabiodl** [ocornut/meka#77]
The games with mapper 11 ( described as : no mapper, ROM up to 48KB (typically 32KB) ), for instance Magical Kid Wiz, appear to be requiring 8Kb of ram. The games are however listed with type SG1. SG1000 has 1Kb ram, SC3000 has 2Kb ram, so they do not work unless the system is emulated with an incorrect amount of ram / the description for mapper 11 is changed / the type for the games is changed .

**Comment by birdybro** [comment]
Magical Kid Wiz requires the unlicensed [8kb ram adapter](https://segaretro.org/8kB_RAM_Adapter) to work and it is an SG-1000 game.

Probably should be a mapper for SG-1000 I would think.

**Comment by fabiodl** [comment]
Indeed. Since in meka.nam mapper 11 seems to be associated with the most part (in all cases?) with such games, it may be a matter of changing the definition of the mapper. 
I guess it is interesting to find out where they were intended to be run on. The system type is SG1, and this suggests SG1000 (+ram extension adapter). However, if they were intended to run on as SMS (as it seems the case for korean games that use legacy video) then the 8 Kb of ram would be provided by the system
